app/views/schools/dashboards/_status.html.erb

```diff
@@ -0,0 +1,31 @@
+<div id="profile-status" class="govuk-grid-row">
+  <div class="<%= status_column_size(type) %>">
```

Comment:
Wondering if the class names view helper could help here? https://www.bigbinary.com/blog/rails-6-1-introduces-class_names-helper

`class_names(type == :disabled ? 'govuk-grid-column-one-third' : 'govuk-grid-column-two-thirds')`